app/services/commits_runner.rb

```diff
@@ -0,0 +1,25 @@
+module CommitsRunner
```

Comment:
Let's just make this a class :) I usually prefer to use modules only if you're including them into other classes.